ui/src/ensembler/components/modal/DeleteEnsemblerModal.js

```diff
@@ -61,7 +61,7 @@ export const DeleteEnsemblerModal = ({
     <ConfirmationModal
       title="Delete Ensembler"
       onCancel={() => modalClosed()}
-      onConfirm={submitForm}
+      onConfirm={(arg) => {submitForm(arg); modalClosed()}}
```

Comment:
This change ensures that modalClosed gets called when the form submission is successful, which resets the confirmation text field:
```js
  const modalClosed = () => {
    setCanDeleteEnsembler(true)
    setEnsemblerUsedByCurrentRouterVersion(false)
    setDeleteConfirmation("")
  }
```